pkg/controller/windowsmachine/windowsmachine_controller.go

```diff
@@ -130,6 +137,12 @@ func add(mgr manager.Manager, r reconcile.Reconciler) error {
 	err = c.Watch(&source.Kind{Type: &core.Node{
 		ObjectMeta: meta.ObjectMeta{Namespace: ""},
 	}}, &handler.EnqueueRequestsFromMapFunc{ToRequests: newNodeToMachineMapper(mgr.GetClient())}, predicate.Funcs{
+		UpdateFunc: func(e event.UpdateEvent) bool {
```

Comment:
Create events need to handled as well, when the operator comes up the existing machines will be given as create events

Reply:
Create events are being handled for node objects. Is your suggestion to handle `create` events for Machine objects as well?

Reply:
Don't you need to check the version annotation in the create events also?